### DISCUSSION TITLE
Support GitHub-managed Terraform state

### DISCUSSION THREAD

**Discussion by F21** [community/community#121306] [Actions]
### Select Topic Area

Product Feedback

### Body

GitLab has the ability to store terraform state in a safe and secure manner: https://docs.gitlab.com/ee/user/infrastructure/iac/terraform_state.html
It would be nice if GitHub Actions has this ability as well.

**Comment by grandmas-favorite** [comment]
Hi @F21!

Thanks for sharing and we appreciate your patience. You can check our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

As a GitHub user, the most you can do is to share your feedback on this Discussion space with the correct tags (include examples), and trust that we are always working to make GitHub as awesome as possible. We can't guarantee that every bit of feedback sees action, but we are paying attention!
